web/docker/Dockerfile

```diff
@@ -0,0 +1,104 @@
+FROM ubuntu:18.04
+
+ARG CC_VERSION=master
+ARG CC_GID=950
```

Comment:
Is the value `950` randomly selected? Is there some recommendation for this on the docker documentation?

Reply:
Yes, the default value is `950`, but you can specify a different one during the build process. I don't know if there is any recommendation in the docker documentation.

Reply:
Standard Unix IDs by convention begin at 1000 by convention. 950 is low enough to still be considered a "daemon user" but high enough not to collide with any names whatsoever in the host user table. The important thing is [here](https://medium.com/@mccode/understanding-how-uid-and-gid-work-in-docker-containers-c37a01d01cf):

> When I examine the processes running outside of the container, I see that it’s mapped to the user “marc”, but inside the container it’s mapped to the user “appuser”. Both of these usernames are just showing the username that **their execution context knows maps to 1001**.

(Highlight my own.)